src/geocat/viz/util/util.py

```diff
@@ -456,33 +547,52 @@ def set_map_boundary(ax,
                   crs=ccrs.PlateCarree())
 
```

Comment:
Not this line but on line 521 (can't add comment there because there is no change between main and this branch):
There is a more consice way to declare 
`if (lon_range[0] >= 0 and
            lon_range[1] <= 0):  # Case when range crosses antimeridian`
which can look like 
`if lon_range[0] >= 0 >= lon_range[1]: ` instead. The changes are:
- No split between line 521 and 522 
- Remove parentheses 
- Improved range declaration for lon data

Not necessarily a detrimental difference, but I just ran across it as a suggestion in PyCharm and think it may be a nice change